### ISSUE TITLE
[Scenario Bundle] Allow users to add sectors without choosing predefined sector devision 

### ISSUE THREAD

**Issue by u-mueller** [OpenEnergyPlatform/oeplatform#1506]
## Description of the issue
If no predefined sector division (SD) is chosen, no sectors are displayed (this has been different some time ago, and maybe has been changed due to my unclear description in issue OpenEnergyPlatform/oeplatform#1524 ). Since users might be unsure, which from the offered SDs their study deals with, they should be given the oppurtunity to at least annotate with the more general sector definitions from the ontology: `agriculture, forestry and land use sector` `energy demand sector` `energy transformation sector` `industry sector` `waste and wastewater sector` (plus maybe even the subsectors of `energy deman sector` in a subsequent hirarchy). This might be better than losing possible entries at this point.

### Factsheet
Study

### Field
S7: Sector divisions
S8: Sectors

## Ideas of solution
1) Add alternative answer "None of the above" 
2) If answer is chosen, display sector classes from the ontology 

`agriculture, forestry and land use sector`
`energy demand sector` ((plus maybe:))

- `building sector`
- `commercial sector`
- `heating and cooling sector`
- `household sector`
- `transport sector`

`energy transformation sector`
`industry sector`
`waste and wastewater sector`

**Comment by jh-RLI** [comment]
@l-emele @stap-m 
Is this topic still relevant? We understand that we should update the sectors that are displayed after selecting "Other" in the sector list. 

Do we still have "only" 3 sector divisions?
![image](https://github.com/OpenEnergyPlatform/oeplatform/assets/38939526/de1d311f-752d-4cc5-8101-c87d49da225d)

**Comment by l-emele** [comment]
No, these are our currently implemented sector division classes:
![grafik](https://github.com/OpenEnergyPlatform/oeplatform/assets/36884905/0cf74cd4-68a9-4ecb-9792-d3e56bfb8018)

Additionally we have the following sector division individuals:
* 'BSI-KritisV sector division'
* 'nace_ sectors'
* 'CRF sectors (IPCC 1996)'
* 'CRF sectors (IPCC 2006)'
* 'KSG sector division'
* 'Eurostat energy balance sector division'
* 'German energy balance sector division'
* 'Renewable Energy Directive sector division'
* 'GovReg sector division'
* 'MMR sector division'

**Comment by stap-m** [comment]
> @l-emele @stap-m
> Is this topic still relevant? We understand that we should update the sectors that are displayed after selecting "Other" in the sector list.

Yes, this is highly relevant. Can you fix this?

**Comment by jh-RLI** [comment]
Yes, we also discussed this at one of the OEKG-DEV meetings. As soon as the oeo-ext module is implemented, I will work on the remaining Urgency-l topics.

**Comment by jh-RLI** [comment]
@adelmemariani

**Comment by stap-m** [comment]
@jh-RLI as discussed in the oekg-dev today: if users select "other" for _sector division_ the options to choose from for _sector_ should be ONLY the sector subclasses, see also my [comment](https://github.com/OpenEnergyPlatform/oeplatform/issues/1506#issuecomment-1859931673).

**Comment by stap-m** [comment]
Still doesn't work. This is REALLY important.

**Comment by jh-RLI** [comment]
Okay, I will look into it today. Not sure if i can fix it quickly.

**Comment by stap-m** [comment]
Sorry for bringing this up again and again for 3 years now, but this is crucial for the usability of the bundles. 
Which the current state, the sectors can not be annotated correctly, which makes the annotation somehow useless.
Is there a reason why this is never being addressed? Too complex?
If this is the case, we NEED to find another technical solution. This is a really important functionality.
@jh-RLI @Ludee

**Comment by jh-RLI** [comment]
Don’t worry about bringing this up again. It’s good that you’re doing so. I’m aware of it too, and it’s on my to-do list. It annoys me as well that I haven’t sorted it out yet.

Let’s arrange a quick meeting next week so we can sort this out. Then you can tell me straight away whether the solution meets the requirements. On the whole, it’s not all that complex (apart from the fact that the code still isn’t particularly easy to maintain – but I can live with that).
@stap-m
